frontend/src/components/UserDetails.vue

```diff
@@ -127,10 +139,25 @@ export default {
       return UsersRepository.getUser(this.userId)
         .then(response => {
           this.user = response.data;
+          // Keycloak returns attributes as arrays which doesn't work with the autocomplete for org details
+          this.user.attributes.org_details = formatOrganization(this.user.attributes.org_details);
+          this.$store.commit("user/setUser", this.user);
         })
         .catch(e => {
           console.log(e);
         });
+    },
+    updateUser: function() {
+      // Validate the User Details
+      if (!this.$refs.form.validate()) {
+        this.$store.commit("alert/setAlert", {
+          message: "Please correct errors before submitting",
+          type: "error"
+        });
+        window.scrollTo(0, 0);
+        return;
+      }
+      this.$emit('submit-user-updates', this.user)
```

Comment:
moved the user details form validation to this method instead of being duplicated in the parent components (update/create user)
this method now emits the user to update or create to the parent components